seawatch_planner/settings/dev.py

```diff
@@ -1,8 +1,9 @@
 import dj_database_url
+import django_heroku
 
 from .base import *
 
-DEBUG = True
```

Comment:
This does not seem right. We could use a third environment just for tests.

Reply:
We don't need a special enviroment because for Live server Tests debug gets always disabled

Reply:
But wouldn't this disable DEBUG mode in the dev environment?

Reply:
sorry i misunderstood your point. Its fixed now